oauthproxy.go

```diff
@@ -6,6 +6,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/authorization"
```

Comment:
This should be included with the other non standard library imports, linter should complain about this